ocaml/typing/solver_intf.mli

```diff
@@ -122,6 +122,18 @@ module type Lattices_mono = sig
      allowed * r". But ocaml doesn't like existentials, and this weaker version
      is good enough for us *)
 
+  (* Usual notion of adjunction f -| g :
+      [f x <= y] iff [x <= g y].
+
+     Our solver accepts a wider notion of adjunction and allows [f] and [g] to
```

Comment:
Let this new statement describe *padjoints* (short for partial adjoint). Then we have:

* The right-padjoint of any constant morphism is a constant morphism returning the maximum element.
* The left-padjoint of any constant morphism is a constant morphism returning the minimum element.

Actually, this definition says that the right-padjoint of any morphism whatsoever is `Const_max`. That doesn't seem right. So I think something's missing here.

Do you agree?